src/lib/components/EventQAndA/index.js

```diff
@@ -0,0 +1,88 @@
+import {BaseElement} from '../BaseElement';
+import './_styles.scss';
+
+/**
+ * Filters out Q&A's based on `data-category` property.
+ *
+ * @extends {BaseElement}
+ * @final
+ */
+class EventQAndA extends BaseElement {
+  constructor() {
+    super();
+    this.selectCategory = this.selectCategory.bind(this);
+
+    this.catageories = new Set();
+    this.categorySelectElement = document.createElement('select');
+    this.categorySelectElement.classList.add('w-select--borderless');
+    this.categorySelectElement.addEventListener('change', this.selectCategory);
+  }
+  connectedCallback() {
+    super.connectedCallback();
+    this.addDropDown();
+
+    this.querySelectorAll('[data-category]').forEach((element) => {
+      element.addEventListener('click', this.closeDetail);
+    });
+  }
+
+  disconnectedCallback() {
+    this.categorySelectElement.removeEventListener(
+      'change',
+      this.selectCategory,
+    );
+
+    this.querySelectorAll('[data-category]').forEach((element) => {
```

Comment:
I'm confused, doesn't `<summary>` and `<details>` handle opening and closing on its own? This looks like we're adding extra JS.